Makefile

```diff
@@ -56,49 +56,41 @@ run-dev-interactive run-bash: | build-dev
 run-cleanup: | build-dev
 	docker run $(dev_args) sh cleanup.sh
 
-run-black: | build-dev
-	docker run $(dev_args) black openslides_backend/ tests/
-
-run-isort: | build-dev
-	docker run $(dev_args) isort --recursive openslides_backend/ tests/
-
-run-flake8: | build-dev
-	docker run $(dev_args) flake8 openslides_backend/ tests/
-
-run-mypy: | build-dev
-	docker run $(dev_args) mypy openslides_backend/ tests/
-
 run-unit-tests: | build-dev
 	docker run $(dev_args) pytest tests/unit/
 
 
-# compose commands
-
-COMPOSE_FILE=docker-compose.dev.yml
+# testing
+# -T is important for GH actions: There is no TTY
 
-run-dev-compose: | build-dev
-	docker-compose -f $(COMPOSE_FILE) up -d
+DC=docker-compose -f docker-compose.dev.yml
 
-run-dev-verbose: | build-dev
-	docker-compose -f $(COMPOSE_FILE) up
+build-tests: | build-dev
+	$(DC) build
 
-stop-dev-compose:
-	docker-compose -f $(COMPOSE_FILE) down --volumes
+start-test-setup: | build-tests
+	$(DC) up -d
+	$(DC) exec -T backend ./wait.sh writer 9011
+	$(DC) exec -T backend ./wait.sh reader 9010
 
-run-compose-bash: | run-dev-compose
-	docker-compose -f $(COMPOSE_FILE) exec backend sh
+start-test-setup-verbose: | build-tests
+	$(DC) up
+	$(DC) exec -T backend ./wait.sh writer 9011
+	$(DC) exec -T backend ./wait.sh reader 9010
 
-run-tests: | run-dev-compose
-	docker-compose -f $(COMPOSE_FILE) exec backend pytest
-	docker-compose -f $(COMPOSE_FILE) down --volumes
+run-tests-interactive: | start-test-setup
+	$(DC) exec backend sh
 
-run-tests-cov: | run-dev-compose
-	docker-compose -f $(COMPOSE_FILE) exec backend pytest --cov
-	docker-compose -f $(COMPOSE_FILE) down --volumes
+run-tests: | start-test-setup
+	$(DC) exec backend pytest
+	$(DC) down --volumes
 
-run-tests-cov-no-tty: | run-dev-compose
```

Comment:
I explicitely added this command so you can still have the normal `run-tests` with TTY and nice colors.